app/src/main/res/common/values-v21/styles.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <style name="AppTheme.TransparentStatusBar" parent="AppTheme.NoActionBar">
+        <item name="android:windowTranslucentStatus">true</item>
+    </style>
+</resources>
```

Comment:
Isn't this the same file as the one in `values-19`?

Reply:
Yes man! my bad 😞